### DISCUSSION TITLE
Is it possible to filter by dynamic value on a relational field?

### DISCUSSION THREAD

**Discussion by Wiermag** [directus/directus#13546] [Archive]
**I have two collections in a data model with a connecting parameter.**
![Zrzut ekranu z 2022-05-25 13-20-08](https://user-images.githubusercontent.com/48072019/170258898-48d2d9c2-8dbd-4f16-ab84-85acaf279085.png)
![Zrzut ekranu z 2022-05-25 13-22-13](https://user-images.githubusercontent.com/48072019/170258947-e0932de0-1415-41fb-bee9-bf4de2bbbb1c.png)

I am trying to configure my relation, but is a problem with primary key - it is not editable when selecting for M2O and M2M fields. In the case of the O2M field it is possible, but then we do not have the correct view and selection of the values we want.

![image](https://user-images.githubusercontent.com/48072019/170258358-ed560c53-43c7-4333-b627-dd18e6173a71.png)

I also tried to filter, but I need possibilities  in M2M/ M2O field to be able to filter on the parent relational dynamic variable between two collections.
I would like to have the value be something like $THIS.storeId, where $THIS is the current product, maybe it could help.

![image](https://user-images.githubusercontent.com/48072019/170258557-01175f9e-ef21-4eef-9523-addc797c1559.png)

Or maybe is another solution to this problem?

**Comment by rijkvanzanten** [comment]
There's currently no way to use the values of the current item in the filter configuration for the m2o interface 👍🏻
